stable/netsil/values.yaml

```diff
@@ -0,0 +1,9 @@
+deployment:
```

Comment:
Group by component: `deployment` -> `streamProcessor`

Reply:
Is this nice to have or a requirement?

Reply:
actually I'm not following what you're asking for.

Reply:
Ah, ok. saw this further down.

Reply:
fixed.